GitUI/UserControls/RevisionGrid/RevisionGridToolTipProvider.cs

```diff
@@ -56,6 +56,15 @@ string GetToolTipText()
                         provider.TryGetToolTip(e, revision, out var toolTip) &&
                         !string.IsNullOrWhiteSpace(toolTip))
                     {
+                        int lineCount = 0;
+                        for (int pos = 0; pos < toolTip.Length; ++pos)
+                        {
+                            if (toolTip[pos] == '\n' && ++lineCount == 30)
+                            {
+                                return toolTip.Substring(0, pos + 1) + ">>> TOOLTIP TRUNCATED <<<";
```

Comment:
Should we use ellipses instead? We don't need to translate them and it is consistent with all other truncations we use.

Reply:
Ellipses should be used consistently. Separate issue and pr?

Reply:
I'm ok with that.